omnibus/Gemfile.lock

```diff
@@ -1,18 +1,19 @@
 GIT
   remote: git://github.com/chef/omnibus-software.git
-  revision: 3e392f27c2a7d49a2aded125d825d54ac191ea1a
+  revision: 8d611676c4458fa679cbc48e2111892ae7986cbf
```

Comment:
Very few default_version bumps in the diff between these two.  Most changes are related to config_guess and license attributes.